integration-tests/bats/sql-shell.bats

```diff
@@ -42,6 +42,13 @@ teardown() {
     [[ "$output" =~ "+---------------------" ]] || false
 }
 
+@test "sql-shell: dolt sql shell has mysql db" {
```

Comment:
Need tests of adding grants via `-q`, listing them again

Reply:
Tests for `sql -q` are in `integration-tests/bats/sql.bats`, but they won't persist right now. They will after my refactor changes though.